src/protocols/bitcoin/views/TransactionDetails.vue

```diff
@@ -72,21 +72,28 @@ export default defineComponent({
 
     const transactionOwner = route.params.transactionOwner as string;
 
+    const bitcoinAdapter = ProtocolAdapterFactory.getAdapter(PROTOCOL_BITCOIN);
```

Comment:
I suggest to rename this to `adapter` as we use only one type of adapters within the system and the `adapter` constant is already more common.